src/commandDetails/coin/info.ts

```diff
@@ -6,25 +6,27 @@ import {
   SapphireMessageExecuteType,
   SapphireMessageResponse
 } from '../../codeyCommand';
+import { getCoinEmoji } from '../../components/emojis';
 import { EMBED_COLOUR } from '../../utils/embeds';
 
 // Get information about coin
+const COIN = getCoinEmoji();
 const infoEmbed = new MessageEmbed()
```

Comment:
My guess as to why "undefined" appeared in the coin info embed is that these constants are initialized before `initEmojis()` in `ready.ts` is completed (I don't currently have my dev environment with me to verify). Would the custom emoji show if you try placing these 2 constants into `coinInfoExecuteCommand` instead?

Reply:
Thanks it worked! The linter's nice

Reply:
Glad it worked! :)